docs/conf.py

```diff
@@ -27,6 +27,9 @@
 project = 'Sphinx Ansible Theme'
 copyright = f'2020 {AUTHOR}'
 
+# URL to send the user to when clicking on the title link
+documentation_home_url = '/'
```

Comment:
Move this to `html_theme_options` or drop because there's already a default.

Reply:
Implemented in 81756b5f0bd428ba3b6e0037f2678ecbe4dc17fa. I think it makes sense mentioning it in this file since this is in docs/.